### DISCUSSION TITLE
Auto deploy preview with Netlify

### DISCUSSION THREAD

**Discussion by hong4rc** [pcottle/learnGitBranching#932] [Q&A] [answered]
I created a pull request https://github.com/hong4rc/learnGitBranching/pull/3

and config

- Build command: `gulp build`
- Publish directory: `build`

![image](https://user-images.githubusercontent.com/19208123/155249889-ea9a6ad1-059f-48ff-ac76-3ee1a20e690b.png)

Quick to detect error on UI

![image](https://user-images.githubusercontent.com/19208123/155249988-56903653-b07f-4794-b6e7-48b9a3fb6376.png)

Do you want to use Netlify for preview, I going to create a pull request to build with Netlify (Netlify need build to a folder instead of `index.html` + `build` + `assets` + `lib` like your `gh-pages`)

**Comment by pcottle** [comment]
Wow it's really neat that you can browse the preview! This would be a nice addition to have.

What if we just build both to a folder for Netlify previews, *and* still build to the `index.html` file? we'll need the index.html file to still be there since that's what https://learngitbranching.js.org/ resolves to

**Reply by hong4rc** [reply]
My commit hong4rc@c2881e1 create `index.html` and copy `assets` + `libs` folder into `build` folder if `process.env.CI` is true. 

if `process.env.CI` is `undefined`, nothing change
![image](https://user-images.githubusercontent.com/19208123/155440939-bc56cc54-04fa-4fad-85a3-068ad8924ec8.png)

**Comment by pcottle** [comment]
Oh yeah that should be an easy way to do it. Excited to see this PR!

**Comment by pcottle** [comment, marked as answer]
#933
